domain_coordinator/setup.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+
+from setuptools import setup
+
+package_name = 'domain_coordinator'
+
+setup(
+    name=package_name,
+    version='0.1.0',
+    description='A tool to coordinate unique ROS_DOMAIN_IDs across multiple processes',
+
+    author='Pete Baughman',
+    author_email='pete.baughman@apex.ai',
+
```

Comment:
I think that it would be good to add `url`, `download_url`, `long_description`, `license`, etc. See https://github.com/ros2/launch/blob/master/launch/setup.py as an example.

Reply:
Sorry for my lazy "etc". `keywords`,  `classifiers` and `install_requires` are missing. The three can be copied from the example.